Sept11.sh

```diff
@@ -0,0 +1,22 @@
+#Script to setup a project directory and download a FASTA file
+#Alan Yu ayu15@dons.usfca.edu
+#September 11, 2017
+
+#set up directories
+echo "Setting up directories..."
+mkdir 2017-09-11_bioinformatics_project
```

Comment:
If you leave this part out, then the script could be used to set up any similar project.